library/flink

```diff
@@ -3,14 +3,14 @@
 Maintainers: The Apache Flink Project <dev@flink.apache.org> (@ApacheFlink)
 GitRepo: https://github.com/apache/flink-docker.git
 
-Tags: 1.16.0-scala_2.12-java8, 1.16-scala_2.12-java8, scala_2.12-java8, 1.16.0-java8, 1.16-java8, java8
+Tags: 1.16.1-scala_2.12-java8, 1.16-scala_2.12-java8, scala_2.12-java8, 1.16.1-java8, 1.16-java8, java8
 Architectures: amd64,arm64v8
-GitCommit: 51ea69cc52e6a759c5b5e463d3920fd7ed8caebc
+GitCommit: 9b0a6a749af9fb2853c93a7aee3429fb13815879
```

Comment:
It looks like the commit id needs to be updated to https://github.com/apache/flink-docker/commit/e348fd602cfe038402aeb574d1956762f4175af0 (the one on master, instead of the commit from the PR branch)

Reply:
That was a stupid mistake, it's fixed now.